README.md

```diff
@@ -1,4 +1,8 @@
-# Mattermost To Do Plugin [![CircleCI](https://circleci.com/gh/mattermost/mattermost-plugin-todo.svg?style=svg)](https://circleci.com/gh/mattermost/mattermost-plugin-todo)
+# Mattermost To Do Plugin 
+
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-todo/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-todo)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-todo/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-todo)
+
```

Comment:
OT: I am a fan of having the HW link at the top of the readme, we should include it in the starter-template, perhaps?

Reply:
@levb Could you name a plugin that does have such link in the readme? I couldn't find one.